tests/src/FunctionalJavascript/Form/AuthenticationFormJsTest.php

```diff
@@ -287,7 +287,7 @@ protected function validateForm(callable $visitFormAsAdmin): void {
     // Test invalid endpoint.
     $invalid_domain = "{$this->randomGenerator->word(16)}.example.com";
     $page->fillField('Apigee Edge endpoint', "http://{$invalid_domain}/");
-    $this->assertSendRequestMessage('.messages--error', "Failed to connect to Apigee Edge. The given endpoint (http://{$invalid_domain}/) is incorrect or something is wrong with the connection. Error message: cURL error 6: Could not resolve host: {$invalid_domain} (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)");
```

Comment:
This is also weird, I am getting the original error on my local that you have just fixed. Which PHP version or CURL lib version you have installed? Maybe this issue also highlights a different problem, we should not test the exceptions returned by an external API or a library.

![image](https://user-images.githubusercontent.com/1755573/57224683-81bad600-700a-11e9-9c1c-b681ac821984.png)

Reply:
OK, It looks like cURL may have changed there message. It's probably not worth digging into since the PR https://github.com/apigee/apigee-edge-drupal/pull/192 truncates the message fixing the issue.